vagrant/README.md

```diff
@@ -1,6 +1,6 @@
 # Vagrant Development Environment
 
-This development environment is designed to allow you to start working quickly and from any OS which can install (Vagrant)[https://www.vagrantup.com/] and (VirtualBox)[https://www.virtualbox.org/wiki/Downloads].
+This development environment is designed to allow you to start working quickly and from any OS which can install [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/wiki/Downloads).
```

Comment:
I'd fix this up into the prior commit, no?

Reply:
Sure. Unfortunately, I didn't catch it prior to committing.

Reply:
Oh, you're suggesting amending the previous commit? Sure.